Dockerfile

```diff
@@ -8,16 +8,14 @@ MAINTAINER Igor Cota <igor@foundationdevices.com>
 
 WORKDIR /root
 
-ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
Does this really help? I thought `ARG` doesn't add layers?

Reply:
Sorry, meant line optimization, not layers!

Reply:
It's just a preference, happy to drop that change.

Reply:
Last thing Dockerfiles need is less lines @sethforprivacy!

Please drop it and I'll merge the rest

Reply:
Done!